### PR TITLE
Remove tp-spacer

### PR DESCRIPTION
**Pull request by tomgenoni**
This removes `functions.scss` which contained our lone `tp-spacer` function. Also removed references to it in the docs.

### CODE REVIEW

File: packages/thumbprint-scss/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Removed
+
+-   [Major] Remove `function.scss` that contained the `tp-spacer` function.
```
Comment (danoc) [inline packages/thumbprint-scss/CHANGELOG.md]: Could you add a note here on how to upgrade? Something along the lines of use `$tp-space-*` Tokens and hardcode values that don't fit in that system.

**Comment by danoc** [comment]
cc: @zboman, @benknight

File: packages/thumbprint-scss/CHANGELOG.md (hunk 1 above)
Comment (tomgenoni) [reply packages/thumbprint-scss/CHANGELOG.md]: Added.

**Comment by tomgenoni** [comment]
Goodnight, sweet prince

**Comment by benknight** [comment]
Looks like `tp-spacer` is not used on the help repo.

**Pull request merged**